### PR TITLE
fix: fix button loading animation appearing on feedback none

### PR DESCRIPTION
**Pull request by dominguesgm**
Fix button loading animation appearing on feedback `none`.

### CODE REVIEW

File: src/components/button/ProgressBar.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ class ProgressBar extends Component {
     }
 
     begin() {
+        console.log('Run begin');
```
Comment (PedroMiguelSS) [inline src/components/button/ProgressBar.js]: Can we remove this `console.log`?

File: src/components/button/ProgressBar.js [hunk 2]
```diff
@@ -48,6 +49,7 @@ class ProgressBar extends Component {
     }
 
     end() {
+        console.log('Run end');
```
Comment (PedroMiguelSS) [inline src/components/button/ProgressBar.js]: Delete this one too.

**Review by PedroMiguelSS — CHANGES_REQUESTED** [review]

File: src/components/button/ProgressBar.js (hunk 2 above)
Comment (dominguesgm) [reply src/components/button/ProgressBar.js]: welp, forgot about this, my bad

**Review by PedroMiguelSS — APPROVED** [review]
LGTM 😎

**Pull request merged**